scipy/special/_special_ufuncs_docs.cpp

```diff
@@ -410,6 +402,83 @@ const char *binom_doc = R"(
     (0.037399983365134115, 0.0)
     )";
 
+const char *cospi_doc = R"(
+    cospi(x, out=None)
+
+    Cosine with `x` multiplied by `pi`.
```

Comment:
@fancidev, "Cosine of `x` multiplied by pi" is ambiguous, and could be interpreted as $\pi cos(x)$; although the latter is probably not likely to appear in natural contexts due to a mismatch of units, I still think it's good to avoid ambiguity.  "Cosine with `x` multiplied by `pi`" is at least unambiguous. Also, strictly speaking, we aren't supposed to use math symbols outside of the Notes section, although we sometimes break that rule, it's typically to add equations where they provide useful context. I don't think using the symbol is needed here. If you think it's important to disambiguate that this is not just the naive calculation `cos(x*pi)`, we could add a sentence. "Numerically advantageous over the direct computation `cos(x*pi)`".